packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-describedby/aria-describedby.docs.mdx

```diff
@@ -0,0 +1,171 @@
+import { Meta, Canvas, Controls, Source } from '@storybook/addon-docs/blocks';
+import * as CCR from './aria-describedby.stories';
+import DefaultExample from './examples/defaultExample.sample.html?raw';
+import DefaultExample2 from './examples/defaultExample2.sample.html?raw';
+import AriaDescribedbyElementsPropertyHTML from './examples/ariadescribedbyElementsProperty.sample.html?raw';
+import AriaDescribedbyElementsProperty from './examples/ariadescribedbyElementsProperty.sample.js?raw';
+import NonWorkingExampleIIa from './examples/nonWorkingExample-IIa.sample.html?raw';
+import NonWorkingExampleIIb from './examples/nonWorkingExample-IIb.sample.html?raw';
+import WorkingExampleIIa from './examples/workingExample-IIa.sample.html?raw';
+import WorkingExampleIIb from './examples/workingExample-IIb.sample.html?raw';
+import NonWorkingExampleIII from './examples/nonWorkingExample-III.sample.html?raw';
+import WorkingExampleIII from './examples/workingExample-III.sample.html?raw';
+import NonWorkingExampleIV from './examples/nonWorkingExample-IV.sample.html?raw';
+import WorkingExampleIV from './examples/workingExample-IV.sample.html?raw';
+
+<Meta of={CCR} />
+
+# aria-describedby
+
+<div className="lead">
+  In the Light DOM, `aria-describedby` is used to give an element a description by referencing the
+  ID of another element or multiple elements that contain the desired text.
+</div>
+<p>
+  This is useful when a simple text label isn't enough or when the description needs to be combined
+  from different pieces of text.
+</p>
+
+### I. Referencing Within the Same DOM Tree
+
+• <b>Both elements in Light DOM</b> ✔️
+
+<Source language="html" dark code={DefaultExample}></Source>
+
+• <b>Both elements in same Shadow DOM</b> ✔️
+
+<Source language="html" dark code={DefaultExample2}></Source>
+<p>
+  <b>
+    The above relationships are valid because in both cases the target element and its description
+    are part of the same tree.
+  </b>
+</p>
+
+<p>
+  Below is a working example of an `aria-describedby`/`id` relationship within the shadowDom of a
+  custom button component. You can <b>test it with NVDA</b> to confirm the correct description
+  announcement.
+</p>
+<Canvas of={CCR.ExampleHTML} sourceState="hide" />
+
+
+### II. Referencing From Outside a Shadow DOM Into That Shadow DOM
```

Comment:
The `aria-describedby` attribute, which is in `shadowdom` references the `id` attribute, which is in the `lightdom`. Therefore either the title or the example is wrong.